chain/vm/mkactor.go

```diff
@@ -73,10 +73,18 @@ func NewBLSAccountActor(st *state.StateTree, addr address.Address) (*types.Actor
 }
 
 func NewSecp256k1AccountActor(st *state.StateTree, addr address.Address) (*types.Actor, aerrors.ActorError) {
+	var acstate account.State
+	acstate.Address = addr
+
+	c, err := st.Store.Put(context.TODO(), &acstate)
+	if err != nil {
+		return nil, aerrors.Escalate(err, "serializing account actor state")
+	}
+
 	nact := &types.Actor{
 		Code:    builtin.AccountActorCodeID,
 		Balance: types.NewInt(0),
-		Head:    EmptyObjectCid,
+		Head:    c,
 	}
```

Comment:
Was this initialization just accidentally missed, or is there a reason not to do it for Secp accts?

Reply:
i think because it wasnt strictly necessary, but this is fine